src/Libraries/Nop.Data/Migrations/UpgradeTo460/DataMigration.cs

```diff
@@ -0,0 +1,169 @@
+﻿using FluentMigrator;
+using Nop.Core.Domain.Customers;
+using Nop.Data.Mapping;
+using Nop.Data.Extensions;
+using Nop.Core.Domain.Directory;
+using Nop.Core.Domain.Localization;
+using Nop.Core.Domain.Tax;
+using System.Data;
+
+namespace Nop.Data.Migrations.UpgradeTo460
+{
+    [NopMigration("2022-01-21 00:00:00", "4.50.0", UpdateMigrationType.Data, MigrationProcessType.Update)]
+    public class DataMigration : Migration
+    {
+        private readonly INopDataProvider _dataProvider;
+
+        public DataMigration(INopDataProvider dataProvider)
+        {
+            _dataProvider = dataProvider;
+        }
+
+        /// <summary>
+        /// Collect the UP migration expressions
+        /// </summary>
+        public override void Up()
+        {
+            // add column
+            var customerTableName = NameCompatibilityManager.GetTableName(typeof(Customer));
+            var firstNameCustomerColumnName = "FirstName";
```

Comment:
Let's use `nameof(Customer.FirstName)` instead of string values, it will be more reliable. Same for all other properties.